js/modules/k6/html/html.go

```diff
@@ -44,17 +48,41 @@ type Selection struct {
 	sel *goquery.Selection
 }
 
-func (s Selection) Add(arg goja.Value) Selection {
-	switch val := arg.Export().(type) {
+func (s Selection) emptySelection() Selection {
+	return s.Eq(s.Size()) //ask for out of bounds item to empty selection
```

Comment:
Comments should be formatted as:

```go
// Ask for out of bounds item for an empty selection.
```

And preferably put before the line, to give you more space for proper sentences.

(Although none of this is admittedly entirely consistent everywhere...)